utils.py

```diff
@@ -161,16 +158,18 @@ def label2data(label, shape, info):
     if "roi" in info:
         for roi in info["roi"]:
             if roi["label"] == label:
-                get_points(data, roi["imagePath"], roi["handles"], shape)
+                get_points(data, roi["imagePath"], roi["handles"], reactOHIF=False)
 
     # for OHIF REACT viewer:
-    elif ("ohifViewer" in info and
-          "measurements" in info["ohifViewer"] and
-          "FreehandRoi" in info["ohifViewer"]["measurements"]):
+    elif (
+        "ohifViewer" in info
+        and "measurements" in info["ohifViewer"]
+        and "FreehandRoi" in info["ohifViewer"]["measurements"]
+    ):
 
```

Comment:
How about an "else" here that logs an error, maybe "unrecognized viewer" or something?  Otherwise it will fail silently.

Reply:
Indeed,
If neither is present, we should indicate here (or earlier) that there is no data we can parse and exit.

NOTE: A WARNING is given earlier in `utils.gather_ROI_info()` when this is the case.  I'd like to warn the user only once on this matter.